### PR TITLE
Packaging - support for Ubuntu 18.04 - update documentation

### PR DESCRIPTION
**Pull request by kowh-ai**
Update the following 2 doco files with Ubuntu 18.04 changes

https://docs.ckan.org/en/latest/maintaining/installing/index.html
https://docs.ckan.org/en/latest/maintaining/installing/install-from-package.html

### Features:

- [ ] includes tests covering changes
- [ ] includes updated documentation
- [ ] includes user-visible changes
- [ ] includes API changes
- [ ] includes bugfix for possible backport

Please [X] all the boxes above that apply

### CODE REVIEW

File: doc/maintaining/installing/index.rst [hunk 1]
```diff
@@ -10,9 +10,9 @@ There are three ways to install CKAN:
 #. Install from Docker Compose
 
 From package is the quickest and easiest way to install CKAN, but it requires
-Ubuntu 16.04 64-bit 64-bit. **You should install CKAN from package if**:
+Ubuntu 14.04, 16.04 or 18.04 64-bit. **You should install CKAN from package if**:
```
Comment (amercader) [inline doc/maintaining/installing/index.rst]: We dropped support for 14.04 on new CKAN versions in f65d992d19625b78fc6ebcde538acf49691ecfbe, as it only has [Extended Maintenance Support](https://ubuntu.com/about/release-cycle). We'll keep providing packages for CKAN versions that had 14.04 packages but not new ones, so let's not mention it here.

TL; DR mention only 16.04 and 18.04

File: doc/maintaining/installing/install-from-package.rst [hunk 2]
```diff
@@ -5,8 +5,8 @@ Installing CKAN from package
 ============================
 
 This section describes how to install CKAN from package. This is the quickest
-and easiest way to install CKAN, but it requires **Ubuntu 16.04 64-bit**. If
-you're not using Ubuntu 16.04 64-bit, or if you're installing CKAN for
+and easiest way to install CKAN, but it requires **Ubuntu 14.04, 16.04 or 18.04 64-bit**. If
```
Comment (amercader) [inline doc/maintaining/installing/install-from-package.rst]: Same as above

File: doc/maintaining/installing/install-from-package.rst [hunk 3]
```diff
@@ -57,7 +57,7 @@ CKAN:
 
 #. Download the CKAN package:
 
-    - On Ubuntu 16.04:
+    - On Ubuntu 18.04:
```
Comment (amercader) [inline doc/maintaining/installing/install-from-package.rst]: Rather than replace the version we need two entries, something like:

```
   - On Ubuntu 16.04:

       .. parsed-literal::

           wget \http://packaging.ckan.org/|latest_package_name_xenial|

   - On Ubuntu 18.04:

       .. parsed-literal::

           wget \http://packaging.ckan.org/|latest_package_name_bionic|


```

The `latest_package_name_bionic` needs defining [here](https://github.com/ckan/ckan/blob/master/doc/conf.py#L310), using `bionic` as distro name.

File: doc/maintaining/installing/install-from-package.rst [hunk 4]
```diff
@@ -66,7 +66,7 @@ CKAN:
 
 #. Install the CKAN package:
 
-   - On Ubuntu 16.04:
+   - On Ubuntu 18.04:
```
Comment (amercader) [inline doc/maintaining/installing/install-from-package.rst]: Same as above, we need to sets of instructions

**Comment by kowh-ai** [comment]
@amercader  - made the changes, hopefully OK now

**Comment by amercader** [comment]
@kowh-ai We are missing the defining of the `latest_package_name_bionic` var in `doc/conf.py` right?

You can try to build the docs locally with `python setup.py build_sphinx` (docs will be built in `build/sphinx/html`)

**Comment by kowh-ai** [comment]
> @kowh-ai We are missing the defining of the `latest_package_name_bionic` var in `doc/conf.py` right?
Actually yes...I'll update that file too
> 
> You can try to build the docs locally with `python setup.py build_sphinx` (docs will be built in `build/sphinx/html`)
I will look into this today and thanks @amercader !

**Comment by Gauravp-NEC** [comment]
> @kowh-ai We are missing the definition of the `latest_package_name_bionic` var in `doc/conf.py` right?
> 
@kowh-ai @amercader , I think issue https://github.com/ckan/ckan/issues/6309 relevant to the missing definition of the `latest_package_name_bionic` var in `doc/conf.py`. Please confirm.
